bumble/pandora/security.py

```diff
@@ -294,23 +294,37 @@ async def Secure(
             try:
                 self.log.debug('Pair...')
 
+                wait_for_security = asyncio.get_running_loop().create_future()
```

Comment:
nit: I may be clearer to use a name that's not an action, but a noun instead. The name `wait_for_security` sounds like it is performing an action, but in fact it is an object that you wait for.  Maybe something like `security_completion` or `security_result`?

Reply:
Done